frontend/src/lib/forms/Button.svelte

```diff
@@ -3,6 +3,7 @@
 </script>
 
 <!-- https://daisyui.com/components/button -->
-<button on:click class="btn btn-primary" class:loading>
+<button on:click class="btn btn-primary">
   <slot />
+  <span class:loading></span>
```

Comment:
I think the way it used to work in daisy was the spinner would be before the text, so just swap the slot and the span around

Reply:
ah thanks, I've updated it